src/bde_copy.cpp

```diff
@@ -1724,17 +1743,26 @@ int main( int argc, char *argv[] )
         return 0;
     }
 
-    if( use_gzip )
+    if( ! output )
     {
+      if( use_gzip )
+      {
```

Comment:
Are you silently discarding the request to use gzip when stdout output is requested here ?
I think if you don't support gzip output with "-" you should exit with an error right after parsing commandline.
If you can support it then please do that, I can imagine valid use cases for it.
But don't silently ignore commandline arguments.

Reply:
I have spent some time trying to support the gzip option with "-", but can't seem to get it to work. I have updated the commandline parsing to exit with an error if both -z and "-" are used in the same command.